src/main/java/com/hillel/items_exchange/model/Image.java

```diff
@@ -15,8 +16,9 @@
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private long id;
-    @Column(name = "resource_url")
-    private String resourceUrl;
+    @Lob
+    @Type(type="org.hibernate.type.BinaryType")
+    private byte[] resource;
```

Comment:
Annotation @Column is not needed for this field?

Reply:
It is not needed when the name of the column into the database is equal to the variable's name.